src/main/reactjs/src/components/publicTranslator/PublicTranslatorFilters.tsx

```diff
@@ -87,23 +100,29 @@ export const PublicTranslatorFilters = ({
               label={t('languagePair.fromPlaceholder')}
               helperText={t('languagePair.fromHelperText')}
               id="filters-from-lang"
-              variant="outlined"
               values={Utils.createMapFromArray(langs.from, t, 'languages')}
               value={filters.fromLang}
+              variant="outlined"
+              filterValue={filters.toLang}
+              primaryOptions={['fi', 'sv']}
               onChange={handleFilterChange('fromLang')}
+              getOptionLabel={getOptionLabel}
```

Comment:
Combobox elementeillä näyttäisi olevan paljon yhteisiä attribuutteja. Voisiko tässäkin tehdä samalla tavalla kuin FillContactDetails.tsx:n getTextBoxAttributes funktiossa?

Reply:
Paketoin nelisen kappaletta samoja attribuutteja

Reply:
Olisiko tämän lopullinen version näin -->

```
  const getComboBoxAttributes = (
    fieldName: keyof PublicTranslatorComboBoxDetails
  ) => ({
    sortByKeys: true,
    autoHighlight: true,
    varian: 'outlined',
    onInputChange: handleComboboxInputChange(fieldName),
    inputValue: inputValues[fieldName],
    value: values[fieldName],
  });
```